styles/App.css

```diff
@@ -1,4 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@600;900&family=Ranchers&family=Secular+One&display=swap');
+/* text on the web (font-weight: 600) */
+/* font-family: 'Raleway', sans-serif;
+/* "enter name" */
+/* font-family: 'Ranchers', cursive; */
+/* header Quiz */
+/* font-family: 'Secular One', sans-serif; */ 
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Raleway', sans-serif;
+    font-weight: 600;
+    color: #FFCB05;
+}
 body {
+    max-height: 100vh;
+    padding: 1.5vh 5vw ;
     font-family: 'Montserrat', sans-serif;
-    margin: 10; /* final version */
 }
+
+.buttonWithText{
+    border-radius: 7vh;
+    background-color: #3762AC;
+    font-size: 3.5vh;
+}
+
+#pokequiz-app{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 1fr 25vh 10vh 28vh;
+}
+
+.firstColumn{
+    grid-column: 1;
+}
+.secondColumn{
+    grid-column: 2;
+}
+
+/* div headerWithLogo */
+
+div#headerWithLogo #pokemonLogo{
+    height: 18vh;
+}
+div#headerWithLogo #quizLogo{
+    padding-left: 15vh;
+    font-family: 'Secular One', sans-serif;
+    font-size: 8.5vh
+}
+
+/* ul selectModeMenu */
+
+ul#selectModeMenu{
+    grid-row: span 2/4;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-top: 5%;
+    padding-left: 6%;
+    list-style:none;
+}
+ul#selectModeMenu li{
```

Comment:
isn't `ul#selectModeMenu li` selector finds the same elements as `.buttonWithText`?

It should be merged together, probably - and better to do it in simpler selector - `.buttonWithText`

Reply:
Yes, but `.buttonWithText` also is relates to button "Play". I grouped the items so as not to repeat the code.